src/login/tests/LoginPage.test.jsx

```diff
@@ -318,6 +307,10 @@ describe('LoginPage', () => {
   });
 
   it('should redirect to social auth provider url on SSO button click', () => {
+    mergeConfig({
```

Comment:
If this is required in most if the test cases, could we move this into `beforeEach()` instead of adding in each test case and disable it for only those where you need to?

Reply:
It is only required in 3 tests out of 24 for LoginPage tests. So, I think moving it to `beforeEach ()` and then disabling it where required would not be a better option.